praetorian_cli/ui/aegis/commands/job.py

```
@@ -226,6 +243,207 @@ def run_job(menu, args):
+            return
+
+        status = job.get('status', 'unknown')
+        capabilities = job.get('capabilities', [])
+        capability = capabilities[0] if capabilities else 'unknown'
+        created = job.get('created', 0)
+        updated = job.get('updated', 0)
+        dns = job.get('dns', '')
+        config = job.get('config', {})
+
+        menu.console.print()
+        menu.console.print(f"  Job Details")
```

Ruff (0.15.6)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)